modules/calendar/html_parser.py

```diff
@@ -19,7 +19,14 @@ def md_links_to_html(text: str) -> str:
 
 
 def html_links_to_md(text: str) -> str:
-	return __HTML_LINK_REGEX.sub(r"[\3\4\6](\2\5)", text)
+	def repl(match):
```

Comment:
Will give false positives for something like `[hw2](https://moodle.jct.ac.il/file/.../compilers-hw2.pdf)`.